unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -149,62 +176,141 @@ export default class ActionBinder {
         },
       },
     };
-    const response = await this.serviceHandler.postCallToService(
-      this.acrobatApiConfig.connectorApiEndPoint,
-      { body: JSON.stringify(cOpts) },
-    );
-    if (!response) return;
-    window.location.href = response.url;
+    this.promiseStack.push(
+      this.serviceHandler.postCallToService(
+        this.acrobatApiConfig.connectorApiEndPoint,
+        { body: JSON.stringify(cOpts) },
+    ));
+    await Promise.all(this.promiseStack)
+    .then((resArr) => {
+      this.progressUpdater(this.splashScreenEl, 100);
+      const response = resArr[resArr.length - 1];
+      if (!response?.url) throw new Error("Error connecting to App");
```

Comment:
I believe wherever we are throwing Errors, we will update to dispatch error event passing in message coming from the authored error spreadsheet. This will be updated in a following PR.

Reply:
@narcis-radu Currently the errors are not handled and would be handled as a part of a different PR that Sanjay would raise.
The throw events are placeholders to some of the errors that would need an error toast